src/interface_r/README.md

```diff
@@ -0,0 +1,36 @@
+## h2o4gpu - R Interface to H2O4GPU
+
+This directory contains the R package for H2O4GPU - a collection of GPU solvers by H2Oai. It builds upon the easy-to-use Scikit-learn API and its well-tested CPU-based algorithms. It can be used as a drop-in replacement for scikit-learn (i.e. import h2o4gpu as sklearn) with support for GPUs on selected (and ever-growing) algorithms. H2O4GPU inherits all the existing scikit-learn algorithms and falls-back to CPU aglorithms when the GPU algorithm does not support an important existing Scikit-learn class option.
```

Comment:
I think you can leave it at `This directory contains the R package for H2O4GPU - a collection of GPU solvers by H2Oai`. The rest is just repeated from the main README.